verifier/src/lib.rs

```diff
@@ -163,22 +181,6 @@ pub trait Verifier<E> {
     /// In order to accommodate constant time operations this returns a
     /// [`VerificationOutput`] instead of a [`Result`].
     fn verify(&self, evidence: &E) -> VerificationOutput<Self::Value>;
```

Comment:
The removal of the `and` and `or` methods is because they didn't actually work.
This is similar to the `VerifierType` in the evidence.rs test. I wasn't able to tweak the signature to get the compiler to infer the resultant type so one had to manually specify the type `verifier.or::<bunch of stuf here>(other)`